src/transaction.js

```diff
@@ -412,6 +431,15 @@ class Transaction {
             this.fee += 37 * feePerByte; // 32 bytes + 5 byte label
         }
     }
+
+    // add the rekey-to field to a transaction not yet having it
+    // supply feePerByte to increment fee accordingly
+    addRekey(reKeyTo, feePerByte=0) {
```

Comment:
why do we add `feePerByte` here? Isn't sizeEstimate happening during the signing process?

Reply:
The final `fee` is calculated at construction, not at signing, so if you mutate the `Transaction` after construction, this function provides the option to update the fee - like we did with `addLease`, above.